pkg/services/ngalert/schedule/schedule.go

```diff
@@ -162,94 +138,22 @@ func (sch *schedule) Unpause() error {
 	return nil
 }
 
-func (sch *schedule) ruleRoutine(grafanaCtx context.Context, key models.AlertRuleKey, evalCh <-chan *evalContext, stopCh <-chan struct{}) error {
```

Comment:
remains unchanged except for the `sch.wg.Done()` on the first line of the function.